homeassistant/components/ring/strings.json

```diff
@@ -35,6 +35,20 @@
     "binary_sensor": {
       "ding": {
         "name": "Ding"
+      },
+      "motion": {
+        "name": "Motion"
+      }
+    },
+    "event": {
+      "ding": {
+        "name": "Ding"
+      },
+      "motion": {
+        "name": "Motion"
+      },
```

Comment:
Doesn't event entities use the device class name if the name is not found?

Reply:
They do, I can drop this and drop the translation key I think

Reply:
Dropped